handlers/main.yml

```diff
@@ -2,6 +2,10 @@
 - name: run riak_disk_tune
   command: /bin/bash /usr/local/bin/riak_disk_tune.sh
 
+- name: restart riak and ensure enabled
+  service: name=riak enabled=yes state=restarted
+  when: riak_init_system == 'system'
```

Comment:
I became concerned with variable scope at execution time of the handler. Do you know if this `riak_init_system` is safe here? I am considering templating the trigger with the variable, and in turn specifying trigger name with `system`.